src/components/AssetListHeader.js

```diff
@@ -12,19 +12,22 @@ const styles = theme => ({
   createButton: {
     backgroundColor: theme.customColors.white,
   },
-  appBarBackground: {backgroundColor: theme.customColors.appBarBackground, boxShadow: theme.shadows[2]}
+  appBar: {
+    backgroundColor: theme.customColors.appBarBackground,
+    paddingLeft: theme.drawerWidth,
+  }
 });
 
 /*
   Renders the app bar of the list view of Assets.
   */
 const AssetListHeader = ({ title, classes }) => (
-  <AppBar position="static" className={classes.appBarBackground}>
+  <AppBar position="fixed" className={classes.appBar}>
     <Toolbar>
       <Typography variant="title" color="inherit" className={classes.flex}>
         { title }
       </Typography>
-      <Link className='App-raised-button-link' to="/asset/create">
+      <Link to="/asset/create">
```

Comment:
Removing this class has introduced a small styling bug. As the button is wrapped in a link (<a />) it attempts to add text decoration. The removed class above looks to remove that (and incorrectly makes the text color black btw). Can we correctly set text-dec to none, and the text color to `coreGreen` like the same primary button on the form.

This was one of the small tasks on my list, but could probably fix in this PR.

Reply:
Sure. Will fix.